src/coreclr/src/debug/di/process.cpp

```diff
@@ -8763,18 +8763,9 @@ CordbAppDomain * CordbProcess::CacheAppDomain(VMPTR_AppDomain vmAppDomain)
     // The cache will take ownership.
     m_appDomains.AddBaseOrThrow(pAppDomain);
 
-    // see if this is the default AppDomain
-    IDacDbiInterface * pDac = m_pProcess->GetDAC();
-    BOOL               fIsDefaultDomain = FALSE;
-
-    fIsDefaultDomain = pDac->IsDefaultDomain(vmAppDomain); // throws
-
-    if (fIsDefaultDomain)
-    {
-        // If this assert fires, then it likely means the target is corrupted.
-        TargetConsistencyCheck(m_pDefaultAppDomain == NULL);
-        m_pDefaultAppDomain = pAppDomain;
-    }
+    // If this assert fires, then it likely means the target is corrupted.
+    TargetConsistencyCheck(m_pDefaultAppDomain == NULL);
+    m_pDefaultAppDomain = pAppDomain;
```

Comment:
`m_pDefaultAppDomain` could be renamed?

Reply:
I think this name is ok. Many other names (including APIs) and comments talk about default appdomain too.